tests/bsim/bluetooth/host/att/long_read/main.c

```diff
@@ -0,0 +1,220 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <argparse.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/settings/settings.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/bluetooth.h>
+
+#include "testlib/adv.h"
+#include "testlib/att_read.h"
+#include "testlib/att_write.h"
+#include "testlib/bs_macro.h"
+#include "testlib/bs_sync.h"
+#include "testlib/conn_ref.h"
+#include "testlib/conn_wait.h"
+#include "testlib/connect.h"
+#include "testlib/log_utils.h"
+#include "testlib/scan.h"
+#include "testlib/security.h"
+
+LOG_MODULE_REGISTER(main, LOG_LEVEL_DBG);
+
+#define UUID_1                                                                                     \
+	BT_UUID_DECLARE_128(0xdb, 0x1f, 0xe2, 0x52, 0xf3, 0xc6, 0x43, 0x66, 0xb3, 0x92, 0x5d,      \
+			    0xc6, 0xe7, 0xc9, 0x59, 0x9d)
+
+#define UUID_2                                                                                     \
+	BT_UUID_DECLARE_128(0x3f, 0xa4, 0x7f, 0x44, 0x2e, 0x2a, 0x43, 0x05, 0xab, 0x38, 0x07,      \
+			    0x8d, 0x16, 0xbf, 0x99, 0xf1)
```

Comment:
It's how clang-format does it. It might do something different if I insert a comment after the 8th byte. I could try but I imagine people won't like the comment.

Reply:
Nah, I've tried to make clang-format make this pretty as well, but can't :D I usually do some manual formatting afterwards to make it better (as clang-format is by no means perfect). 

It's ok as is